api_app/db/repositories/shared_services.py

```diff
@@ -26,11 +26,11 @@ def shared_service_query(shared_service_id: str):
 
     @staticmethod
     def active_shared_services_query():
-        return f'SELECT * FROM c WHERE {IS_ACTIVE_CLAUSE} AND c.resourceType = "{ResourceType.SharedService}"'
+        return f'SELECT * FROM c WHERE {IS_NOT_DELETED_CLAUSE} AND c.resourceType = "{ResourceType.SharedService}"'
 
     @staticmethod
-    def active_shared_service_with_template_name_query(template_name: str):
-        return f'SELECT * FROM c WHERE {IS_ACTIVE_CLAUSE} AND c.resourceType = "{ResourceType.SharedService}" AND c.templateName = "{template_name}"'
+    def deployed_shared_service_with_template_name_query(template_name: str):
```

Comment:
I guess we're not testing this, as I can't see an associated change in a test.
Not something you need to address...just an observation.

Reply:
Well there is a test `test_create_shared_service_item_with_the_same_name_twice_fails` here: https://github.com/microsoft/AzureTRE/blob/main/api_app/tests_ma/test_db/test_repositories/test_shared_service_repository.py#L107 

It checks that if the query returns existing items, then the shared service isn't created again. 

I wanted to change that test so that I can set the actual items in the container, and then run a query on it, but I haven't found an easy way to do that

Reply:
In azure/cosmos/container.py, there's a method that queries items, and a query is passed in there. So basically there's no way to mock the items it's going to use but supply query from the method being tested.

We could consider using a local Cosmos emulator to be able to write these kinds of tests: https://docs.microsoft.com/en-us/azure/cosmos-db/local-emulator?tabs=ssl-netstd21

Reply:
Leave things as they are for now.
It was just something I spotted while reviewing this, but sounds like there are good reasons (it's non-trivial)